gui/api/resources.py

```diff
@@ -4095,3 +4110,269 @@ def dehydrate(self, bundle):
                 'state': state
             })
         return bundle
+
+
+class JsonUser(object):
+    def __init__(self, id, name, label):
+        self.id = id
+        self.name = name
+        self.label = label
+
+
+class JsonUserResource(DojoResource):
+
+    id = fields.CharField(attribute='id')
+    name = fields.CharField(attribute='name')
+    label = fields.CharField(attribute='label')
+
+    class Meta:
+        allowed_methods = ['get']
+        resource_name = 'account/json_users'
```

Comment:
Perhaps we can come up with something better for that. "account/all_users" or something?